scripts/util/configurations.py

```diff
@@ -30,6 +30,7 @@ class GymEnvironmentConfig(EnvironmentConfig):
     gym: str = Field(description="Gym Environment <openai, dmcs, pyboy>")
     task: str
     domain: Optional[str] = ""
+    display: Optional[int] = 0
```

Comment:
Is there a reason for int over boolean here?

Reply:
It is the issue we had where the boolean basically just means if the parameter is given or not - if you do bool and then --display False it will actually be True because it exists. The bool in parse-args is non-intuitive - plus future-proofing the idea of displaying training vs evaluation etc